server/app.js

```diff
@@ -0,0 +1,19 @@
+require('env2')('./.env');
+const { join } = require('path');
+const express = require('express');
+const logger = require('morgan');
+
+const app = express();
+
+app.set('port', process.env.PORT || 8080);
+app.use(
+  [express.json(), express.urlencoded({ extended: false })],
+  express.static(join(__dirname, '..', 'client', 'public')),
```

Comment:
on development its usless to handel static file , 
but on production we need to handle build folder not public 
**fix folder name** to
express.static(join(__dirname, '..', 'client', 'build'))